### DISCUSSION TITLE
https://simeonat.github.io/blog/git-python-scripts/

### DISCUSSION THREAD

**Discussion by giscus** [SimeonAT/simeonat.github.io#1] [General]
#https://simeonat.github.io/blog/git-python-scripts/

Although Github is amazing, I find it really fun to self-host my own Git server. It also provides peace of mind, as all of my Git repositories are in a server that I have ownership of (rather than being just on Github).

https://simeonat.github.io/blog/git-python-scripts/

**Comment by SimeonAT** [comment]
If you have any feedback and suggestions on how I can improve my backup implementation, feel free to leave a comment! I will even add it to the "Improvements" section of this blog post.

**Comment by pamandat** [comment]
Thank you for this wonderful tutorial. Me and my chicken learned so much, especially since he is pursuing a degree in cluck-user science. My chicken hopes that next time, in addition to Python, you will come out with a tutorial for the language “Chickums.”

**Comment by Ansh-Kumar** [comment]
I love the implementation of it. Hopefully you can find a much simpler way to migrate all the repos through Gitea software updates - it seems kinda pointless to hold the old repos on local machine and the remote...Maybe you could contribute to the Gitea Open Source :)

**Reply by SimeonAT** [reply]
Thanks Ansh! What you're saying is in the correct direction too, as [Shawn Armstrong](https://www.linkedin.com/feed/update/urn:li:activity:7086199530244751360/?commentUrn=urn%3Ali%3Acomment%3A%28activity%3A7086199530244751360%2C7086220960198004736%29) mentioned that there's probably a much simpler solution too.
